### PR TITLE
feat: add flag to reload mock config file

### PR DESCRIPTION
**Pull request by yuluo-yx**
- For #409

### CODE REVIEW

File: atest
Comment (LinuxSuRen) [inline atest:1]: Forget to delete?

File: cmd/mock.go [hunk 1]
```diff
@@ -17,17 +17,26 @@ limitations under the License.
 package cmd
 
 import (
+	"fmt"
 	"os"
 	"os/signal"
 	"syscall"
 
+	"github.com/hpcloud/tail"
```
Comment (LinuxSuRen) [inline cmd/mock.go:25]: The last commit was 6 years ago. Please consider a well-maintained library, or you can even implement it yourself. Such as https://stackoverflow.com/questions/8270441/go-language-how-detect-file-changing

File: atest
Comment (yuluo-yx) [reply atest:1]: yep, It's just a draft for now, I'll fix it when it goes to review status.

File: cmd/mock.go (hunk 1 above)
Comment (yuluo-yx) [reply cmd/mock.go:25]: got it.

File: atest
Comment (LinuxSuRen) [reply atest:1]: you can add it into the `.gitignore`

**Pull request closed without merging**